include/MidiJack.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * MidiJack.h - Jack MIDI client
+ *
+ * by Shane Ambler
+ *
+ * This file is part of Linux MultiMedia Studio - http://lmms.sourceforge.net
```

Comment:
I think this should be edited to match all our other file descriptions. [Example here.](https://github.com/LMMS/lmms/blob/master/src/core/midi/MidiAlsaRaw.cpp#L1-L23)